src/impact-dashboard/CurveChartContainer.tsx

```diff
@@ -20,8 +23,12 @@ function combinePopulations(data: CurveData, columnIndex: number) {
   ).map(([incarcerated, staff]) => incarcerated + staff);
 }
 
-const CurveChartContainer: React.FC<Props> = ({ markColors }) => {
-  const modelData = useEpidemicModelState();
+const CurveChartContainer: React.FC<Props> = ({ chartHeight, markColors, modelData }) => {
+  // Temporarily appeasing the linter
```

Comment:
Obviously this needs to go, but first we have to figure out how to pass state around.  See the conversation here:  https://github.com/Recidiviz/covid19-dashboard/pull/117#issuecomment-614934105

Reply:
I actually think your proposal/wondering in the last paragraph of that comment sounds right, and I think you're on the right path as-is. Having a `FacilityContext` wrapping each facility on the page, each containing an `EpidemicModelContext` that maintains its own state that is being persisted to the backend via the `FacilityContext` feels clean. There would be a list of `FacilityContexts` in the page, ordered by whatever our ordering scheme is (facility name?), and when we go through the flow to add a new facility, we would initialize a new `FacilityContext` with a new `EpidemicModelContext` that has default parameters auto-populated.

Does that sound reasonable to you, @sprad and @macfarlandian?

Reply:
I think that's on the right track ... one thing that comes to mind is that `EpidemicModelContext` will have to stop managing the URL state, because when multiples are on the page they will start fighting over it.

Reply:
Ok, this approach makes sense to me.  I don't know a ton about Contexts/Providers, etc., but I'll start diving in...

Reply:
@macfarlandian great point. I think with this change we probably _have_ to get rid of the URL parameter updates. It's possible that generating a URL with those parameters that can be shared would still be a valid use case, but I'm not sure if @cawarren has spoken on this topic.